code/upaya/app/assets/javascripts/application.js

```diff
@@ -14,6 +14,8 @@
 //= require jquery_ujs
 //= require jquery-ui
 //= require turbolinks
+//= require_tree ../../../vendor/assets/javascripts
```

Comment:
This, and adding jquery to vendors, are not necessary. We're already using the `jquery-rails` gem, which provides jQuery. https://github.com/rails/jquery-rails#installation